docs/fundamentals/app-host-overview.md

```diff
@@ -141,6 +141,15 @@ For more information, see [GPU support in Podman](https://github.com/containers/
 
 The preceding code adds a container resource named "ollama" with the image "ollama/ollama". The container resource is configured with multiple bind mounts, a named HTTP endpoint, an entrypoint that resolves to Unix shell script, and container run arguments with the <xref:Aspire.Hosting.ContainerResourceBuilderExtensions.WithContainerRuntimeArgs%2A> method.
 
+#### Container resource lifecycle
+
+When the app host is run, the container resource is used to determine what container image to create and start. Under the hood, .NET Aspire runs the container using the defined container image by delegating calls to the appropriate OCI-compliant container runtime, either Docker or Podman. The following commands are used:
```

Comment:
I don't totally understand, does this make it clearer by splitting up the sentence?

```suggestion
When the app host is run, the container resource is used to determine what container image to create and start. Under the hood, .NET Aspire runs the container using the defined container image. The image is selected by delegating calls to the appropriate OCI-compliant container runtime, either Docker or Podman. The following commands are used:
```

Reply:
I'm not sure how to clarify that more. I'm trying to explain that when the .NET Aspire app host runs, it manages all container lifecycles on your behalf. For example, when you're adding something like Redis or RabbitMQ, it legit calls `docker container create` and then `docker container start` on the image used by those integrations.